### PR TITLE
New user here

### PR DESCRIPTION
**Pull request by genesis331**

### CODE REVIEW

File: humans/genesis331.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>genesis331</title>
+        <meta charset="utf-8">
+    </head>
+    <body>
+    </body>
```
Comment (SubhrajyotiSen) [inline humans/genesis331.html]: It would be great if you could add some content to your HTML page 😃

**Review by SubhrajyotiSen — CHANGES_REQUESTED** [review]

File: humans/genesis331.html (hunk 1 above)
Comment (genesis331) [reply humans/genesis331.html]: Of course!!

**Comment by genesis331** [comment]
Done!!

**Comment by genesis331** [comment]
Btw can you teach me how to publish Github Pages on <username>.github.io instead of <username>.github.io/<repository>??

**Comment by genesis331** [comment]
Btw can you teach me how to publish Github Pages on username.github.io instead of username.github.io/repository??

**Review by SubhrajyotiSen — APPROVED** [review]

**Pull request merged**